cadc-util/src/main/java/ca/nrc/cadc/auth/OpenIdPrincipal.java

```diff
@@ -35,33 +35,44 @@
 package ca.nrc.cadc.auth;
 
 import java.io.Serializable;
+import java.net.URL;
 import java.security.Principal;
 
 /**
- * Class that represents an openID identity.
+ * Class that represents an openID identity. The principal consists of an immutable
+ * open ID and its corresponding issuer.
  */
 public class OpenIdPrincipal implements Principal, Serializable {
-    private static final long serialVersionUID = 20140625143750L;
+    private static final long serialVersionUID = 202407041230L;
 
     private String openID;
+    private URL issuer;
 
     /**
      * Ctor
      * 
      * @param openID
      */
-    public OpenIdPrincipal(final String openID) {
+    public OpenIdPrincipal(final URL issuer, final String openID) {
```

Comment:
javadoc: should have the issuer param

Should we make it more clear in the field name `openID` that this is the `sub` (subject) field?